tools/free_energy/alchemical_analysis/alchemical_analysis.xml

```diff
@@ -0,0 +1,95 @@
+<tool id="alchemical_analysis" name="Alchemical Analysis" version="1.0.2">
+    <description>Analysis of alchemical free energy calculations</description> 
+    <requirements>
+         <requirement type="package" version="1.0.2">alchemical-analysis</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+       ln -s '$datain' ./data.tar &&
+       tar xf ./data.tar && 
+       mkdir ./RESULTS/ &&
+            #if $lambda.skip_lambda == "no"
+                alchemical_analysis -d data -o ./RESULTS -t '$temp' -s '$time' -c -q xvg -p '$prefix' -g -e -f 10 -w -u '$units' &>> '$output'
+            #else if $lambda.skip_lambda == "yes"
+                 alchemical_analysis -d data -o ./RESULTS -t '$temp' -s '$time' -k '$lambda.skiplambda' -c -q xvg -p '$prefix' -g -e -f 10 -w -u '$units' &>> '$output'
+            #end if
+    ]]></command>
+    <inputs>
+    <param format="tar" name="datain" type="data" label="TI/FEP data output"/>
```

Comment:
a collection of files?

multiple="True"

Reply:
@bgruening @simonbray The tools need one directory with all the XVG files and I need to explicitly specify the path to that file and also uploading hundreds of files maybe not convenient, user errors can happen and also histories will look very complex. Generally, we run multiple simulations for one ligand until we get the proper convergence. That's why I thought to use a tar file and keep the inputs simple. If this is not a good idea we can change the input file format.

Reply:
It's really a matter of the amount of data, if its hundreds of files or even thousands, let's go for a tarball at the moment. We could also think about to make it a proper filetype. xvg.tar or something like that. There is no usecase where a user wants to "view"/observe one given file (in such a case)?

In general, I think Galaxy should support the use-case of tar -> collection during upload, but this is another story.

Reply:
No no need to view each and every file. Making an xvg.tar file type is a great idea. I will think about that.

Reply:
Personally I think a .xvg.tar filetype is overkill. Packing xvg files into tarballs is not a general usecase IMO.

I also doubt there will be hundreds of xvg files present. When I did TI simulations I had 10 or 20 data points. I guess more than 50 would be quite unusual. So from that point of view a collection should be quite feasible.